integ/test_http.py

```diff
@@ -0,0 +1,30 @@
+"""
+Testing the http sink
+"""
+import json
+from mock import patch
+
```

Comment:
The python instance that runs the tests didn't have requests installed. Added the lib to .travis.yml